packages/shopify-codemod/README.md

```diff
@@ -5,9 +5,10 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 ## Usage
 
 1. `npm install -g jscodeshift`
-1. `git clone https://github.com/shopify/javascript` or [download the zip file](https://github.com/shopify/javascript/archive/master.zip)
-1. `jscodeshift -t <codemod-script> <file>`
-1. Use the `-d` option for a dry-run and use `-p` to print the output for comparison.
+2. `git clone https://github.com/shopify/javascript` or [download the zip file](https://github.com/shopify/javascript/archive/master.zip)
+3. `npm install && npm run setup`
```

Comment:
It would be fine if people did install `jscodeshift` locally, It's just easier to show it work when installed globally (since the executable is part of the path).